src/test/scala/com/codergaden/ex1/ApplicationSpec.scala

```diff
@@ -0,0 +1,32 @@
+package com.codergaden.ex1
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class ApplicationSpec extends AnyFlatSpec with Matchers {
+
+  "The getShorterAndLonger function with an empty list" should "get a Result containing 2 equal WordsWithLength with empty list each " in {

```

Comment:
you can remove the word function (that actually is a method), just leave "getShorterAndLonger called with empty list